conf/messages

```diff
@@ -231,15 +231,18 @@ error.max.length.ba.software.package.name = Names of the software packages must
 error.invalid.characters.ba.software.package.name = Names of the software packages can include letters a to z, numbers, spaces, and most punctuation and special characters
 error.required.ba.select.nca = Select yes if your business is registered with the National Crime Agency
 error.required.ba.option.risk.assessment = Select yes if your business has a documented risk assessment policy and procedure
-error.required.ba.risk.assessment.format = Select how you document your risk assessment policy and procedure
-error.required.ba.business.use.accountant = Select yes if your business receives professional advice on Money Laundering Regulations and due diligence
-error.required.ba.suspicious.activity = Select yes if you have written guidance on how to identify and report suspicious activity
-error.required.ba.tax.matters = Select yes if {0} manages your tax affairs with HMRC
+error.required.ba.risk.assessment.format = Select one or more options to show what formats you use for your risk assessment policy and procedure
+error.required.ba.business.use.accountant = Select yes if your business receives advice on Money Laundering regulations and due diligence
+error.required.ba.suspicious.activity = Select yes if your business has written guidance for staff on how to report suspicious activity
+error.required.ba.tax.matters = Select yes if your professional adviser also manages your tax affairs with HMRC
+
```

Comment:
Were the content changes above (lines 234-237) part of this ticket?

Reply:
I think something went awry with a merge somewhere, I've corrected these.